effekt/shared/src/main/scala/effekt/core/PrettyPrinter.scala

```diff
@@ -59,9 +60,17 @@ object PrettyPrinter extends ParenPrettyPrinter {
     vsep(definitions map toDoc, semi)
 
   def toDoc(e: Extern): Doc = e match {
-    case Extern.Def(id, tps, cps, vps, bps, ret, capt, body) =>
-      "extern" <+> toDoc(capt) <+> "def" <+> toDoc(id) <+> "=" <+> paramsToDoc(tps, vps, bps) <> ":" <+> toDoc(ret) <+> "=" <+> toDoc(body)
-    case Extern.Include(contents) => emptyDoc // right now, do not print includes.
+    case Extern.Def(id, tps, cps, vps, bps, ret, capt, bodies) =>
+      "extern" <+> toDoc(capt) <+> "def" <+> toDoc(id) <+> "=" <+> paramsToDoc(tps, vps, bps) <> ":" <+> toDoc(ret) <+> "=" <+> vcat(bodies map {
+        case ExternBody.StringExternBody(ff, body) => toDoc(ff) <+> toDoc(body)
+        case ExternBody.EffektExternBody(ff, body) => toDoc(ff) <+> "(" <+> toDoc(body) <+> ")"
+      })
+    case Extern.Include(ff, contents) => emptyDoc // right now, do not print includes.
+  }
+
+  def toDoc(ff: FeatureFlag): Doc = ff match {
+    case FeatureFlag.NamedFeatureFlag(name) => name
+    case FeatureFlag.Default => "else"
```

Comment:
Nice, here it is already called `Default` :)

Reply:
Just noticed so too when renaming 😅